api.yaml

```diff
@@ -85,6 +85,75 @@ paths:
             #! if eq .Response.Body.errorType "InvalidInput" !# 400
             #! else !# 500
             #! end !#, "bodyPassthrough": true}'
+  /v1/cloud/asset:
+    get:
+      summary: "Retrieve a list of known cloud assets at point in time"
+      parameters:
+        - name: "time"
+          in: "query"
+          description: "The point in time details for asset list"
+          required: true
+          schema:
+            type: "string"
+            format: "date-time"
+        - name: "count"
+          in: "query"
+          description: "Number of cloud assets to return"
+          required: true
+          schema:
+            type: "integer"
+            minimum: 1
+        - name: "offset"
+          in: "query"
+          description: "Offset of the first cloud asset to return for paged API access. Zero-based."
+          required: false
+          schema:
+            type: "integer"
```

Comment:
suggestion: A numeric offset exposes the use of a SQL datastore as an implementation detail. If there's another marker we could use, such as the timestamp of the last element we returned, then it would help the API be agnostic of the datastore.

Reply:
ACK. Good idea!